src/screens/Lending/Dashboard/InfoModalBottom.js

```diff
@@ -11,7 +11,7 @@ import TrackScreen from "../../../analytics/TrackScreen";
 
 type button = {
   title: string,
-  onPress: function,
+  onPress: Function,
```

Comment:
preferably we should specify exactly the signature btw, because `Function` is like `any` function. but ok for now.